index.bs

```diff
@@ -571,9 +571,9 @@ The sample code for generating and registering a new key follows:
       },
 
       timeout: 360000,  // 6 minutes
-      excludeCredentials: [], // No exclude list of PKCredDescriptors
-      extensions: {"loc": true}  // Include location information
-                                               // in attestation
+      excludeCredentials: [{"id": h'EFE752BFC60C74E131D53052674E7113F41C44C538D70BA24077EE8F79F6DCEEC51F9BCACBBA95CCCE88AF5A1D2D04CE5A0E2F0F19C9C503177B5994495771AB', "type": "public-key"}, 
+	  {"id": h'9D99ED41E9894E8055EC99ABEB015E4314F21711C8D905C388079E4D3F18EB3D24672953FA2AC7015B2155BCE8C040878D7256EACAA8A24E6F1462DE6314516E', "type": "public-key"}], // Exclude list of PKCredDescriptors
+      extensions: {"appidExclude": true}  // Check the exclude list aginst U2F appid
```

Comment:
Extension input needs to be the AppID to test against. Also improved indentation a bit.

```suggestion
      excludeCredentials: [
        // Don't re-register any authenticator that has one of these credentials
        {"id": h'EFE752BFC60C74E131D53052674E7113F41C44C538D70BA24077EE8F79F6DCEEC51F9BCACBBA95CCCE88AF5A1D2D04CE5A0E2F0F19C9C503177B5994495771AB', "type": "public-key"},
        {"id": h'9D99ED41E9894E8055EC99ABEB015E4314F21711C8D905C388079E4D3F18EB3D24672953FA2AC7015B2155BCE8C040878D7256EACAA8A24E6F1462DE6314516E', "type": "public-key"}
      ],
      
      // Make excludeCredentials check backwards compatible with credentials registered with U2F
      extensions: {"appidExclude": "https://acme.example.com"}
```